DesignSafe-CI/portal
# Task/des 2294  versioning frontend pipeline field research

## Overview: ##
Add versioning option for FR project types.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2294](https://jira.tacc.utexas.edu/browse/DES-2294)

## Summary of Changes: ##
Added FR Versioning page
Updated Citation page (Author Order) to work for FR and Exp

## Testing Steps: ##
1. Go to FR project --> Publication Preview
2. Click Publish/Version/Amend button --> Click Version
3. On Version Selection page, user should see selected Documents & Missions that have been published. User should have option to select other Documents & Missions that have not been published. These appear collapsed. When uncollapsed, author, descriptions, etc should show along with included files. --> Continue
4. On the confirm Author Order page, Document/Mission appear correctly depending on type. When author is saved, the success message shows correct Document/Mission. 

## UI Photos:

## Notes: ##
